vid/src/avid_m/config.rs

```diff
@@ -29,52 +25,6 @@ pub trait AvidMConfig {
     fn raw_share_digest(raw_shares: &[Self::BaseField]) -> VidResult<Self::Digest>;
 }
 
```

Comment:
@mrain ping to check if deleting this code is fine or we should keep it. It seems to be unused.